roles/confluent.variables/defaults/main.yml

```diff
@@ -406,7 +409,10 @@ kafka_broker_jmxexporter_enabled: "{{jmxexporter_enabled}}"
 ### Port to expose prometheus metrics. Beware of port collisions if colocating components on same host
 kafka_broker_jmxexporter_port: 8080
 
+## destination path for broker jmx config file
 kafka_broker_jmxexporter_config_path: /opt/prometheus/kafka.yml
+## source file for broker jmx config file
+kafka_broker_jmxexporter_config_file: "kafka.yml"
```

Comment:
please match zookeeper_jmxexporter_config_source_path

Reply:
Also if you don't mind, can you make this change across all the other components?

Reply:
Branch updated